frontend/src/components/user-info-panel/user-info-panel.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC } from 'react';
+
+import arrowIcon from '@assets/images/arrow-right-left.svg';
+import balanceIcon from '@assets/images/compare.svg';
+import eyeIcon from '@assets/images/eye.svg';
+import heartIcon from '@assets/images/heart.svg';
+
+import styles from './styles.module.scss';
+
+const UserInfoPanel: FC = () => {
+  const likedItCount = 1;
+  const comparisonCount = 5;
+  const comparedCount = 3;
+  const viewedCout = 2;
+  return (
+    <ul className={styles.UserPanel}>
+      <li className={styles.UserPanelItem}>
+        <img className={styles.UserPanelIcon} src={heartIcon} alt="liked it" />
+        <a className={styles.UserPanelLink} href="#">
+          Liked it
+        </a>
```

Comment:
We must use React Router `Link` components. Because plain `a` tags will reload the page on click

Reply:
@oleksandr-danylchenko These links lead to sections inside the page. In this case, the page does not restart

Reply:
@oleksandr-danylchenko I changed this tag. Please check pull request